### DISCUSSION TITLE
How to identify the root cause of error : z2m: No converter available for 'action'

### DISCUSSION THREAD

**Discussion by jglafkid** [Koenkk/zigbee2mqtt#23632] [General]
Hi, 
I get that error in z2M and no way to identify the root cause. no name of device and i have 84 devices. Nothing more in logs
Any idea

Error 2024-08-16 09:55:07z2m: No converter available for 'action' ("")

Info :
Version de Zigbee2MQTT : [1.39.1](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.39.1)
commit: [e132316a : (https://github.com/Koenkk/zigbee2mqtt/commit/e132316a)
Type de coordinateur. : ConBee2/RaspBee2
Révision du coordinateur. : 0x26780700
Version de l'interface : 0.7.4
Version Zigbee-herdsman-converters : 19.72.0
Version Zigbee-herdsman : 0.55.3
Statistiques : Total 84

**Comment by jglafkid** [comment]
Somebody has an idea ?

**Comment by chris-1243** [comment]
It would be great to know which device throws this error.
Your device is trying to call "action" but there is no converter available for such command.

Also, any automations wrongly coded?

**Comment by jglafkid** [comment]
the problem is that there is no information about the device that cause this error ! i have 83 devices so impossible to check manually. Is there a way to trace because debug is not more informative that what i putted in description ! Le 2 nov. 2024 à 17:50, chris-1243 ***@***.***> a écrit :﻿
It would be great to know which device throws this error.
Your device is trying to call "action" but there is no converter available for such command.
Also, any automations wrongly coded?
